testing/features/forms/input.feature

```diff
@@ -35,16 +35,6 @@ Feature: Input forms
       When I type "Hello" into the text box
       Then the text box has a value of "Hello"
       And there is a label
-      And there is a hint
-      And there is an optional label
-      But there is no error message
-
-    Scenario: Welsh Input can have an optional label
-      Given the language is Welsh
-      When I type "Hello" into the text box
-      Then the text box has a value of "Hello"
-      And there is a label
-      And there is a hint
```

Comment:
given they are of the same speed, can we leave all properties it shows. So for this one can we leave `And there is a hint` *Unless we're changing the physical property*

Reply:
The diff is a bit mad - basically I:
 - removed the Welsh test as it's in the rspec now
 - remove the hint and the optional label from the error example as it's not rendered with them anymore (covered by the texts above tho still)

Reply:
So we've actually changed the properties of a couple of these forms then?

Reply:
Yeah - the optional text is only added on the optional example (it was in error example too before), the hint text is only added in the hint example and fixed width example (it was in error example too before).